challenges/objects-arrays.js

```diff
@@ -99,23 +156,40 @@ The zoos need a list of all their animal's names (animal_name only) converted to
 
 */
 
-const lowCaseAnimalNames = [];
+
+const lowCaseAnimalNames = zooAnimals.map(animal => {
+
+  return animal.animal_name.toLocaleLowerCase();
+  
+});
+
+
```

Comment:
Good work here moving the variable initialization and combining it with your map function!